src/command_modules/azure-cli-backup/azure/cli/command_modules/backup/custom.py

```diff
@@ -0,0 +1,755 @@
+        result = client.revoke(vault_name, resource_group_name, fabric_name, container_uri, item_uri, rp_name,
+                               raw=True, custom_headers=_get_custom_headers())
+        _track_backup_operation(resource_group_name, result, vault_name)
+
+
+def disable_protection(client, resource_group_name, vault_name, container_name, item_name,  # pylint: disable=unused-argument
+                       container_type="AzureIaasVM", item_type="VM", delete_backup_data=False, **kwargs):
+    item = show_item(backup_protected_items_cf(None), resource_group_name, vault_name, container_name, item_name,
+                     container_type, item_type)
+
+    # Get container and item URIs
+    container_uri = _get_protection_container_uri_from_id(item.id)
+    item_uri = _get_protected_item_uri_from_id(item.id)
+
+    # Trigger disable protection and wait for completion
+    if delete_backup_data:
+        result = client.delete(vault_name, resource_group_name, fabric_name, container_uri, item_uri, raw=True,
+                               custom_headers=_get_custom_headers())
+        return _track_backup_job(result, vault_name, resource_group_name)
+
+    vm_item = _get_disable_protection_request(item)
+
+    result = client.create_or_update(vault_name, resource_group_name, fabric_name, container_uri, item_uri, vm_item,
+                                     raw=True, custom_headers=_get_custom_headers())
+    return _track_backup_job(result, vault_name, resource_group_name)
+
+
+def list_jobs(client, resource_group_name, vault_name, status=None, operation=None, start_date=None, end_date=None):
+    query_end_date, query_start_date = _get_query_dates(end_date, start_date)
+
+    filter_string = _get_filter_string({
+        'status': status,
+        'operation': operation,
+        'startTime': query_start_date,
+        'endTime': query_end_date})
+
+    return _get_list_from_paged_response(client.list(vault_name, resource_group_name, filter_string,
+                                                     custom_headers=_get_custom_headers()))
+
+
+def show_job(client, resource_group_name, vault_name, name):
+    return client.get(vault_name, resource_group_name, name, custom_headers=_get_custom_headers())
+
+
+def stop_job(client, resource_group_name, vault_name, name):
+    client.trigger(vault_name, resource_group_name, name, custom_headers=_get_custom_headers())
+
+
+def wait_for_job(client, resource_group_name, vault_name, name, timeout=None):
+    start_timestamp = datetime.utcnow()
+    job_details = client.get(vault_name, resource_group_name, name, custom_headers=_get_custom_headers())
+    while _job_in_progress(job_details.properties.status):
+        if timeout:
+            elapsed_time = datetime.utcnow() - start_timestamp
+            if elapsed_time.seconds > timeout:
+                break
+        job_details = client.get(vault_name, resource_group_name, name, custom_headers=_get_custom_headers())
+        time.sleep(30)
```

Comment:
Consider showing a warning if the command times out.
Also, it doesn't look like this command returns anything. Do you want to return `job_details` once done?

Reply:
Will add a warning for the timing out scenario. Yes, returning job_details sounds like a good idea - will try to do this.